### DISCUSSION TITLE
ZEPETO Studio Unity 파일 폴더 3.1.11.1 문제

### DISCUSSION THREAD

**Discussion by Lumis015** [naverz/zepeto-studio-kor#341] [5. Q&A]
이 문제를 만난 사람이 있습니까? 아니면 저만 있습니까?
이제 Unity 2020.3.9 및 FileFolder 3.1.11.1(2022.01.27 업데이트)을 사용합니다.
FBX를 ZPT 형식의 파일로 내보내면 이렇게 5MB씩 커져서 샵에 올릴 수 없어서(최대 1MB) 혼란스럽습니다.

이전 버전에는 이전에 이 문제가 없었기 때문에 아무도 나에게 대답할 수 있습니까?정말 감사합니다!

**(저는 구글 번역기를 사용하기 때문에 약간 이상할 수 있습니다)**

**Comment by Zepeto-TechEvangelist** [comment]
#322 It's the same as the issue.
please check if the problem still occurs in the latest version.
